external/kafka/src/main/scala/org/apache/gearpump/streaming/kafka/lib/source/AbstractKafkaSource.scala

```diff
@@ -92,6 +91,7 @@ abstract class AbstractKafkaSource(
 
     LOG.info("KafkaSource opened at start time {}", startTime)
     this.startTime = startTime
```

Comment:
Is startTime still needed if change `maybeRecover()` to `maybeRecover(startTime)`?